pkg/render/manager.go

```diff
@@ -434,6 +439,7 @@ func (c *managerComponent) managerEnvVars() []corev1.EnvVar {
 		{Name: "CNX_PROMETHEUS_API_URL", Value: fmt.Sprintf("/api/v1/namespaces/%s/services/calico-node-prometheus:9090/proxy/api/v1", common.TigeraPrometheusNamespace)},
 		{Name: "CNX_COMPLIANCE_REPORTS_API_URL", Value: "/compliance/reports"},
 		{Name: "CNX_QUERY_API_URL", Value: "/api/v1/namespaces/calico-system/services/https:calico-api:8080/proxy"},
+		{Name: "CNX_DASHBOARD_API_URL", Value: "/dashboard"},
```

Comment:
The best practice for RESTful APIs is to use plural nouns for resource names in the URL path, so I'd propose `/dashboards`.